i18n/en.ts

```diff
@@ -217,7 +217,136 @@ const text: Translations = {
+        audioVideoCenterText:
+          'The library has a fully air-conditioned audiovisual centre for seminars, conferences, guest lectures, user awareness programs, etc. with a seating capacity of 100 participants. It is also equipped with a videoconferencing facility.',
+        jGatePlusText:
+          'J-Gate Custom Content for Consortium (JCCC) is a virtual library of journal literature created as a customized e-journals access gateway and database solution. It acts as a one-point access to 7900+ journals subscribed currently under UGC INFONET Digital library consortium as well as university libraries designated as Inter Library Loan (ILL) Centers besides index to open access journals.',
+        nptelText:
+          'The Library has procured NPTEL Web & Video Courses designed & developed by IIT, Chennai in various discipline of Engineering & Sciences for the use of Faculty Members, Research Scholars and Students. Users can access these video courses through Library storage server: ',
+        remoteAccessText:
+          'To provide the off-campus access to subscribed e-resources, the library has subscribed to the KNIMBUS service. The users can create their account either by visiting the URL nitkkr.knimbus.com or by writing to us at librarian@nitkkr.ac.in. After creating the account, the users can log in and access all the e-resources from anywhere.',
+        antiPlagiarismText:
+          'The library has subscribed to anti-plagiarism software Turnitin for all the Faculty Members, Research Scholars and Students. The users can check the plagiarism of their research papers, articles, theses, dissertations, etc. using this facility.',
+      },
+      quickLinks: {
+        collectionResources: 'Collection & Resources',
+        libraryCommittee: 'Library Committee',
+        membershipPrivileges: 'Membership Privileges',
+      },
+      contactUs: {
+        name: 'Name',
+        designation: 'Designation & Qualification',
+        phoneNumber: 'Phone Number',
+        email: 'Email',
+      },
+      libraryCommittee: {
+        libraryCommitteeTitle: 'Library Committee',
+        srNo: 'Sr. No.',
+        name: 'Name',
+        generalDesignation: 'General Designation',
+        libraryCommitteeDesignation: 'Library Committee Designation',
+      },
+      CollectionAndResources: {
+        title: 'Collection & Resources',
+        totalDocuments: 'TOTAL DOCUMENTS',
+        noOfDocuments: '1,72,237',
+        totalBooks: 'LIBRARY BOOKS',
+        noOfBooks: '54,325',
+        bookBank: 'Book Bank',
+        backSets: 'Back Sets',
+        standards: 'Standards',
+        cdsDvds: 'CDs/DVDs',
+        eBooks: 'e-Books',
+        thesis: 'Thesis',
+        noOfBookBank: '81,259',
+        noOfBackSets: '7,097',
+        noOfStandards: '10,097',
+        noOfCdsDvds: '832',
+        noOfEBooks: '12,272',
+        noOfThesis: '6,355',
+        eresources: 'E-Resources',
+        Eresources: {
```

Comment:
The way I've done capitalisation is that the fields which show up in a new "page" are capitalised, the rest are lowercase. So keep that in mind